test/auth.test.js

```diff
@@ -58,15 +67,57 @@ describe('Auth', () => {
     return client.login('email', 'password')
       .then(userId => expect(userId).toEqual('fake-user-id'));
   });
+});
 
-  it('should return a promise for login with only existing auth data userId', async() => {
-    window.fetch.resolves(mockApiResponse());
-    expect.assertions(1);
+describe('Auth login semantics', () => {
+  const email = 'test_user@domain.com';
+  const password = 'password';
+  let test = new StitchMongoFixture();
+  let th;
+  let client;
 
-    let client = new StitchClient();
-    await client.auth.storage.set(common.USER_AUTH_KEY, mockAuthData());
+  beforeAll(async() => await test.setup());
+  afterAll(async() => await test.teardown());
```

Comment:
Same question to all uses of before* and after* where it's a single line return

Reply:
fixing

Reply:
Apparently, that isn't allowed.

Reply:
why not?

Reply:
```
encountered a declaration exception
TypeError: fn.call is not a function
      
      at Object.asyncFn (node_modules/jest-jasmine2/build/jasmine-async.js:38:30)
          at Promise (<anonymous>)
      at Promise.resolve.then.el (node_modules/p-map/index.js:46:16)
          at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)
```

It's expecting to be wrapped in a block by the looks of it

Reply:
what code are you passing it?

Reply:
`beforeAll(test.setup())` / the equivalent for each block

Reply:
darn. oh well